content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,207 @@
+# Flare Architecture
+
+This section contains information about some parts of the Flare Network.
+In addition, some information regarding Avalanche Platform is also present, since Flare adopted some parts of the Avalanche in it's own ecosystem.
```

Comment:
```suggestion
In addition, it also contains some information regarding the Avalanche platform, since Flare adopted some of its parts within its own ecosystem.
```

WDYT?